### PR TITLE
add geocube

### PR DESCRIPTION
**Pull request by ocefpaf**
See https://pypi.org/project/geocube/

@snowman2 is is OK to add this recipe and you as a maintainer?

### CODE REVIEW

**Comment by snowman2** [comment]
Yes, this is great, thanks!

File: recipes/geocube/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/g/geocube/geocube-{{ version }}.tar.gz
+  sha256: ef74e677510ab9ef17bb855d65c187fbd50098a5697effdd4749b9cd5c516d79
+
+build:
+  number: 0
+  noarch: python
+  # entry_points:
```
Comment (chrisburr) [inline recipes/geocube/meta.yaml]: Is this commented out for a reason? (It's the cause of the build failure)

**Comment by snowman2** [comment]
Any updates on this? Anything needed from my end?

File: recipes/geocube/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    - python
+    - pip
+  run:
+    - appdirs
+    - click >=6.0
+    - datacube >=1.6.2,<1.7
+    - geopandas
+    - python
+    - rasterio
+    - scipy
```
Comment (snowman2) [inline recipes/geocube/meta.yaml]: scipy can be replaced with rioxarray for the current version (0.0.3).

Comment (ocefpaf) [reply recipes/geocube/meta.yaml]: Done.

File: recipes/geocube/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipes/geocube/meta.yaml]: > Any updates on this? Anything needed from my end?

Not really. I will leave the CLI stuff commented out until we get a new release. Thanks!

Comment (ocefpaf) [reply recipes/geocube/meta.yaml]: > I'm 90% sure they're already in this release as this Windows failure is normally caused by not declaring entrypoints.

@chrisburr the entrypoint was declared in the setup.py but the module does not exist. That was already updated upstream and we are waiting on a new release but it is safe to merge the `noarch` now.

**Comment by snowman2** [comment]
Thanks!

**Pull request merged**